create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+# Gitops stack using DOKS and Flux CD
 
+This blueprint will guide you step by step on how to spin up a single DOKS (DigitalOcean Kubernetes) cluster and Flux CD for managing application deployments in a GitOps fashion. In the end we will also tell [Flux CD](https://fluxcd.io) to perform a basic deployment of the now ubiquitous `busybox` Docker application.
+
+[Terraform](https://www.terraform.io) was chosen to write `infrastructure as code` using declarative configuration files which allows for concise descriptions of resources using blocks, arguments, and expressions. In our guide it will be responsible with spinning up the DOKS (DigitalOcean Kubernetes) cluster as well as [Flux CD](https://fluxcd.io).
+
+[Flux CD](https://fluxcd.io) is used for managing the `Continuous Delivery` of applications inside the DigitalOcean Kubernetes cluster as well as a `Gitops` enabler.
+
+
+## Bootstrapping DOKS and Flux CD
+
+This section contains information about how we can bootstrap **DOKS** and **Flux CD** via **Terraform** using **Github** as a SCM provider and source of truth. The bootstrap process will create for us a DOKS cluster alongside with a new Github repository and branch where all the manifests used to provision [Flux CD](https://fluxcd.io) will be stored. The same repository will be used to store our Kubernetes custom application deployment manifests that will be managed automatically via [Flux CD](https://fluxcd.io).
```

Comment:
```suggestion
This section contains information about how we can bootstrap DOKS and Flux via Terraform using GitHub as a SCM provider and source of truth. The bootstrap process will create a DOKS cluster alongside with a new Github repository and branch where all the manifests used to provision Flux will be stored. The same repository will be used to store our Kubernetes custom application deployment manifests that will be managed automatically via Flux.
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.